app/controllers/account_recovery_setup_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AccountRecoverySetupController < ApplicationController
+  include UserAuthenticator
+
+  before_action :confirm_two_factor_authenticated
+
+  def index
+    return redirect_to account_url if user_piv_cac_and_phone_enabled?
```

Comment:
I wondered if we'd rather have this be a test of multiple 2fa options, but this should be good for now since we only get here through the piv/cac setup. Longer term, this may be a good hook to prompt someone to have multiple 2fa options configured.